fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CancelCompactionStmtAnalyzer.java

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.sql.analyzer;
+
+import com.starrocks.analysis.BinaryPredicate;
+import com.starrocks.analysis.BinaryType;
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.IntLiteral;
+import com.starrocks.analysis.SlotRef;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.ast.AstVisitor;
+import com.starrocks.sql.ast.CancelCompactionStmt;
+
+public class CancelCompactionStmtAnalyzer {
+
+    public static void analyze(CancelCompactionStmt statement, ConnectContext context) {
+        new CancelCompactionStmtAnalyzerVisitor().analyze(statement, context);
+    }
+
+    static class CancelCompactionStmtAnalyzerVisitor extends AstVisitor<Void, ConnectContext> {
+
+        private long txnId;
```

Comment:
`txnId` is only used in `analyzeWhereClause` function, so `txnId` as class member is not necessary?